charts/redis/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: redis
```

Comment:
redis chart should be a dependency of hyperflow chart
there is no need to create a redis chart, there is one created by redis already out there
remove this chart and use helm dependency management